controllers/fundsController.js

```
@@ -6,22 +6,67 @@ function fundsController() {
   // Post  a FUnd
   const createFund = async (req, res) => {
     const value = req.body;
+
+    // Validate data types
+    if (typeof value.money !== 'number' || isNaN(value.money)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)